### PR TITLE
lmb-1194: Denderleeuw fix duplciate mandatarissen

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

Remove the duplicates

## How to test

See if there are any duplicsates in the Gemeente IV and double check the start date

### CODE REVIEW

File: config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      ?mandataris astreams:formerType mandaat:Mandataris.
+
+      ?membership a astreams:Tombstone.
+      ?membership dcterms:modified ?now .
+      ?membership astreams:deleted ?now .
+      ?membership astreams:formerType org:Membership .
+    }
+  }
+ WHERE {
+    VALUES (?mandataris ?persoon) {
```
Comment (Rahien) [inline config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql:26]: remove person from query, it doesn't serve a purpose

File: config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      ( <http://data.lblod.info/id/mandatarissen/8e590c0d-8016-473f-b990-c03bd7197e5a> <http://data.lblod.info/id/personen/af09a92e521b3833dabc64d901cf62b66b4d653a2b381fe023a559b3bd754a5d> ) # Yves - De Smet
+      ( <http://data.lblod.info/id/mandatarissen/5ff1bac1-ba64-4a82-91a0-c8d9c72fa5f0> <http://data.lblod.info/id/personen/3b4692bf-e5aa-4a74-8c70-7f1fb18e663e> ) # Chancelvie - Okitokandjo # 2026!!
+    }
+
+      GRAPH <http://mu.semte.ch/graphs/organizations/856cbc8af3fbe7390d43420e249c288596e87fbc069c601751d5c5fcd52c543b/LoketLB-mandaatGebruiker> {
+      ?mandataris mandaat:isBestuurlijkeAliasVan ?persoon.
+      ?mandataris org:hasMembership ?membership.
+      ?mandataris ?p ?o.
+      OPTIONAL {
+        ?mandataris dcterms:modified ?modified.
```
Comment (Rahien) [inline config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql:55]: this optional doesn't matter. you're removing everything from the mandataris anyway

File: config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      ( <http://data.lblod.info/id/mandatarissen/eff229ca-b021-49f5-97a7-36ff949f4f05> <http://data.lblod.info/id/personen/3b4692bf-e5aa-4a74-8c70-7f1fb18e663e> ) # Chancelvie - Okitokandjo
+      ( <http://data.lblod.info/id/mandatarissen/53482b56-848b-4e8f-a18b-f4bd348d7b88> <http://data.lblod.info/id/personen/af09a92e521b3833dabc64d901cf62b66b4d653a2b381fe023a559b3bd754a5d> ) # Yves - De Smet
+      # Samenstelling Vast Bureau Denderleeuw
+      ( <http://data.lblod.info/id/mandatarissen/8e590c0d-8016-473f-b990-c03bd7197e5a> <http://data.lblod.info/id/personen/af09a92e521b3833dabc64d901cf62b66b4d653a2b381fe023a559b3bd754a5d> ) # Yves - De Smet
+      ( <http://data.lblod.info/id/mandatarissen/5ff1bac1-ba64-4a82-91a0-c8d9c72fa5f0> <http://data.lblod.info/id/personen/3b4692bf-e5aa-4a74-8c70-7f1fb18e663e> ) # Chancelvie - Okitokandjo # 2026!!
+    }
+
+      GRAPH <http://mu.semte.ch/graphs/organizations/856cbc8af3fbe7390d43420e249c288596e87fbc069c601751d5c5fcd52c543b/LoketLB-mandaatGebruiker> {
+      ?mandataris mandaat:isBestuurlijkeAliasVan ?persoon.
+      ?mandataris org:hasMembership ?membership.
```
Comment (Rahien) [inline config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql:52]: would make this optional to be safe

File: config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql [hunk 4]
```diff
@@ -0,0 +1,59 @@
+  PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#>
+  PREFIX ext: <http://mu.semte.ch/vocabularies/ext/>
+  PREFIX dcterms: <http://purl.org/dc/terms/>
+  PREFIX astreams: <http://www.w3.org/ns/activitystreams#>
+  PREFIX org: <http://www.w3.org/ns/org#>
+
+  DELETE {
+    GRAPH <http://mu.semte.ch/graphs/organizations/856cbc8af3fbe7390d43420e249c288596e87fbc069c601751d5c5fcd52c543b/LoketLB-mandaatGebruiker> {
+      ?mandataris ?p ?o.
```
Comment (Rahien) [inline config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql:9]: you're not deleting the triples from the membership itself. needs ?membership ?mp ?mo.

File: config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql (hunk 1 above)
Comment (JonasVanHoof) [reply config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql]: Removed

File: config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql (hunk 2 above)
Comment (JonasVanHoof) [reply config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql]: Removed, indeed

File: config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql (hunk 4 above)
Comment (JonasVanHoof) [reply config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql]: Deleting ?mp and ?mo now

**Review by Rahien — APPROVED** [review]

**Pull request merged**